build-tools/scripts/BuildEverything.mk

```diff
@@ -80,8 +80,6 @@ _MSBUILD_ARGS	= \
 	/p:AndroidSupportedHostJitAbis=$(call join-with,:,$(ALL_HOST_ABIS)) \
 	/p:AndroidSupportedTargetAotAbis=$(call join-with,:,$(ALL_AOT_ABIS))
 
-CONFIGURATIONS ?= Debug Release
-
 .PHONY: leeroy jenkins leeroy-all opentk-jcw framework-assemblies
```

Comment:
The build failed because the `leeroy` rule was building both configurations before and Jenkins uses `jenkins` rule, which depends on `leeroy`.

So we need to either let `jenkins` rule to run `leeroy` twice or change the Jenkins configuration to run `make jenkins ...` twice.

Now the Jenkins build ran make in this order:
```
+ make prepare-deps V=1 'MSBUILD_ARGS=/p:AutoProvision=True /p:AutoProvisionUsesSudo=True /p:IgnoreMaxMonoVersion=False'
+ make prepare jenkins V=1 'MSBUILD_ARGS=/p:AutoProvision=True /p:AutoProvisionUsesSudo=True /p:IgnoreMaxMonoVersion=False'
+ make all-tests V=1
+ make all-tests CONFIGURATION=Release V=1
```
So the tests in the Release configuration were ran without making `prepare` and `all` in Release configuration first.

Reply:
Yes, we'll need to update Jenkins to run `make leeroy` twice, once per configuration value.

@radekdoulik: Thank you, I hadn't considered that problem. Doh!